mono/metadata/assembly.c

```diff
@@ -310,6 +339,7 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	FACADE_ASSEMBLY ("System.Xml.XmlSerializer"),
 	FACADE_ASSEMBLY ("System.Xml.Xsl.Primitives"),
 	{"WindowsBase", 3},
+	{"cscompmgd", 0},
```

Comment:
I don't think we should add assemblies we know we'll never add/support

Reply:
@marek-safar https://github.com/mono/mono/blob/master/mcs/class/Cscompmgd/Makefile 😄

Reply:
This was just one example, there are more like System.Web.RegularExpressions etc

Reply:
I only added assemblies which we're building as part of net_4_x. I think we should add them all for consistency regardless of whether they are "useful".